src/IJ_mv/HYPRE_IJMatrix.c

```diff
@@ -1011,7 +1011,14 @@ HYPRE_IJMatrixGetValuesAndZeroOut( HYPRE_IJMatrix matrix,
 
    if (exec == HYPRE_EXEC_DEVICE)
    {
-      hypre_error_w_msg(HYPRE_ERROR_GENERIC, "HYPRE_IJMatrixGetValues not implemented for GPUs!");
+      /* TODO: add device implementation */
```

Comment:
Do the non-sstruct changes here (IJ, parcsr, etc.) match what's on master?  Just curious.